### ISSUE TITLE
Upload images

### ISSUE THREAD

**Issue by dinushay** [WAppAI/assistant#116]
Bing has the opportunity to recognize pictures

**Comment by veigamann** [comment]
There's an open [pull request](https://github.com/waylaidwanderer/node-chatgpt-api/pull/452) that might add this feature to [node-chatgpt-api](https://github.com/waylaidwanderer/node-chatgpt-api). If it's approved, we're considering adding it to Sydney. For now, it's not on my immediate list unless that pull request gets merged. While node-chatgpt-api already supports image creation, we haven't brought it into Sydney yet. If you're up for it, we're open to contributions. I will leave this open until that PR gets merged (or not).

**Comment by veigamann** [comment]
Update: This has been added on commit 3e6bbf7

It's another branch I've been working on lately. A full refactor. Many improvements. There's no audio transcription there yet, but it's being worked on atm.

I'd encourage you to switch branches and give it a try, however, since it's not merged into the master branch yet, don't expect much support for new issues. All feedback is appreciated.

If you decide to give it a try, switch to pnpm-refactor, make sure you're on the latest commit, then run `pnpm generate` and `pnpm start`. Also take a good look at your .env properly before starting.
